src/pages/signupOne.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useState } from 'react';
+//import { useNavigate } from 'react-router-dom';
+
+export default function Login() {
+   const [agreeCheck, setAgreeCheck] = useState([false, false]);
+   //const navigate = useNavigate();
+
+   const handleCheckboxChange = (index: number) => {
+      const newAgreeCheck = [...agreeCheck];
+      newAgreeCheck[index] = !newAgreeCheck[index];
+      setAgreeCheck(newAgreeCheck);
+   };
+
+   const nexthandle = () => {
+      // 체크 여부 확인
+      if (agreeCheck[0] === true && agreeCheck[1] === true && agreeCheck[2] === true) {
```

Comment:
`if(agreeCheck[0] === true)`는 `if(agreeCheck[0])`과 동일하죠!

```suggestion
      if (agreeCheck[0] && agreeCheck[1] && agreeCheck[2]) {
```